### ISSUE TITLE
Portrait videos are saved rotated (possible fix solution included!)

### ISSUE THREAD

**Issue by lindalevanen** [afollestad/material-camera#168]
Hello,

Landscape videos are saved correctly oriented on device but portrait front cam videos are saved rotated 90 degrees cw and back cam videos 90 degrees ccw. This is possibly why it has a warning not to take portrait videos/photos?

It would be a huge help to know in onActivityResult how the video was taken so I can rotate it later if needed! 

Thank you in advance, it's such a great library, would suck to change to using another one.

**Comment by tylerkrupicka** [comment]
Yeah rotation can be device specific, and the manufacturers usually account for it in their camera app implementation. There's some rotation logic specific to samsung in the codebase. It's possible this issue would just require similar logic added to your device, would you mind posting the device id? Thanks for the bug report @lindalevanen.
